ouroboros-consensus/src/Ouroboros/Consensus/Node/ProtocolInfo/Byron.hs

```diff
@@ -69,7 +110,7 @@ protocolInfoByron (NumCoreNodes numCoreNodes) nid params gc mLeader =
                       -- TODO: make pbftSignatureThreshold default to a value computed from pbftNumNodes,
                       --       but configurable?
                     }
-                , pbftIsLeader = proofOfCredentials nid <$> mLeader
+                , pbftIsLeader = proofOfCredentials delegation <$> mLeader
```

Comment:
We don't need to pass this, it's already available in scope.

Reply:
Done.